### PR TITLE
#51 - Debian.Asto fully implemented

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #51 
Fully implemented `Debian.Asto` and added tests.

### CODE REVIEW

File: src/test/java/com/artipie/debian/DebianTest.java [hunk 1]
```diff
@@ -118,10 +131,47 @@ void addsPackagesIndex() throws IOException {
...
         );
+        final Key release = this.debian.generateRelease().toCompletableFuture().join();
+        final ListOf<String> substrings = new ListOf<>(
+            String.format("Codename: %s", DebianTest.NAME),
+            "Architectures: amd64",
+            "Components: main",
+            "Date:",
+            "SHA256:",
+            "my_deb_repo/binary/amd64/Packages.gz",
+            "my_deb_repo/binary/amd64/Packages"
```
Comment (genryxy) [inline src/test/java/com/artipie/debian/DebianTest.java:142]: @olenagerasimova these strings meet twice. I'd suggest to extract them

**Review by genryxy — CHANGES_REQUESTED** [review]
@olenagerasimova thanks, one suggestion

**Comment by olenagerasimova** [comment]
@genryxy thanks, corrected

**Review by genryxy — APPROVED** [review]
@olenagerasimova thanks

**Pull request merged**